test/cli/samples/watch/watch-event-hooks/_config.js

```diff
@@ -0,0 +1,24 @@
+const { assertIncludes } = require('../../../../utils.js');
+
+module.exports = {
+	description: 'onStart event hoot shell command executes correctly',
+	command:
+		'rollup -cw --watch --watch.onStart "echo start" --watch.onBundleStart "echo bundleStart" --watch.onBundleEnd "echo bundleEnd" --watch.onEnd "echo onEnd" --watch.onError "echo onError"',
+	stderr(stderr) {
+		assertIncludes(
+			stderr,
+			`watch.onStart $ echo start
+start
+bundles main.js → _actual...
+watch.onBundleStart $ echo bundleStart
+bundleStart
+created _actual in 16ms
```

Comment:
A CLI test must not depend on things like runtime. The usual solution is to use `assertIncludes` for substrings like `created _actual` etc.

Reply:
Another idea: Split the output by line, filtering empty ones and mapping the remaining ones to their first word. Then you would just need to compare

```
start
bundles
bundleStart
created
...
```